src/network/networkd-network.c

```diff
@@ -382,6 +382,8 @@ int network_load_one(Manager *manager, OrderedHashmap **networks, const char *fi
                 .dhcp6_pd_manage_temporary_address = true,
                 .dhcp6_pd_subnet_id = -1,
 
+                .dhcp_server_bind_to_interface = true,
+
```

Comment:
Please drop the blank line. Could you also drop the blank line above the line `.dhcp_server_emit_router = true;`?